ext/xmlrpc/xmlrpc-epi-php.c

```diff
@@ -744,42 +785,31 @@ PHP_FUNCTION(xmlrpc_server_create)
 	}
 
 	if (USED_RET()) {
```

Comment:
Do we really need check? If not, can the other usages be removed (from array.c for example)?

Reply:
In this case, the USED_RET() check makes no sense. The ones in array.c are present because the functions both have a side-effect and a return value, and the check saves computing the return value if it isn't used.

Reply:
I got rid of all the `USED_RET()` checks